packages/truffle-artifactor/package.json

```diff
@@ -12,7 +12,7 @@
   },
   "dependencies": {
     "fs-extra": "6.0.1",
-    "lodash": "4.17.11",
+    "lodash": ">=4.17.13",
```

Comment:
`>=` is dangerous because this will automatically include `5.0.0`, which might have breaking changes